src/components/summary.json

```diff
@@ -137,11 +137,11 @@
            {
              "headers": {
                "Drug Description": "Name",
-               "Quantity": { "key": "Quantity", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": false},
+               "Quantity": { "key": "Quantity", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": true},
                "Written Date": { "key": "Start", "formatter": "datishFormat" },
                "Dispensed": { "key": "End", "formatter": "datishFormat" },
-               "Prescriber": { "key": "Prescriber", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": false},
-               "Pharmacy": { "key": "Pharmacy", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": false}
+               "Prescriber": { "key": "Prescriber", "formatter": "stringSubstitutionFormat", "formatterArguments": [""], "sortable": true},
```

Comment:
I **think** you won't need the formatter here anymore, i.e. `{ "key": "Prescriber", "sortable": true}` the formatter function, `stringSubstitutionFormat`, was used previously to display empty string since we don't have the field yet.

Reply:
I didn't know what that did. Thanks!